packages/access-policy/src/index.js

```diff
@@ -107,4 +107,15 @@ export class AccessPolicy {
     }
     return this.cachedPermissionGroupSets[cacheKey];
   }
+
+  /**
+   * Return entities the user has access to the given permission group for
+   *
+   * @param {string} [permissionsGroup]
+   *
+   * @returns entities[] The entity objects
+   */
+  getEntitiesByPermission(permissionGroup) {
```

Comment:
```suggestion
  getEntitiesAllowed(permissionGroup) {
```
For consistency with all the 'allow' terminology above, and we also have a bit of a convention that `getXByY` returns `{ y1: [x1, x3], y2: [x2]}` rather than taking `y1` and returning `[x1, x3]`
Please also add this to the readme of access policy - it's used by a bunch of internal packages, so the docs can come in handy to show what is publicly accessible

Reply:
Renamed and added to readme